drape_frontend/my_position_controller.cpp

```diff
@@ -459,16 +448,7 @@ void MyPositionController::OnLocationUpdate(location::GpsInfo const & info, bool
 
   if (!m_isPositionAssigned)
   {
-    // If the position was never assigned, the new mode will be the desired one except next cases:
     location::EMyPositionMode newMode = m_desiredInitMode;
-    if (m_mode == location::NotFollowNoPosition)
```

Comment:
Put ASSERT(m_mode != location::NotFollowNoPosition, ());
Just in case ..

Reply:
I am not sure here, but let's try and see if it fires.

Reply:
Added